.github/workflows/macos.yml

```
@@ -15,127 +19,55 @@ permissions:
-        with:
-          repository: theory/pgtap
-          path: ./pgTapExtension/
-
-      - name: Install pgTAP
-        if: steps.cache-pgtap.outputs.cache-hit != 'true'
-        run: |
-          cd ./pgTapExtension/
-          make -j 4
-          sudo make install
-          cpan TAP::Parser::SourceHandler::pgTAP
+          brew install postgresql@14 postgis pgrouting 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)